resources/templates/md/posts/2016-04-04-state-management.md

```diff
@@ -0,0 +1,364 @@
+{:title "Clojure state management"
+:description  "How we can manage states in Clojure (how data might change over time), or llamas love candies too :)"
+:layout :post
+:tags  ["Clojure" "ClojureScript" "State management"]
```

Comment:
No need for a "State management" tag, you can use the already existing "Fundamentals"